config/config.go

```diff
@@ -66,9 +66,12 @@ func Load(file string) (config ProjectConfig, err error) {
 
 func fix(config ProjectConfig) ProjectConfig {
 	if len(config.Files) == 0 {
-		config.Files = []string{
-			"README.md",
-			"LICENSE.md",
+		config.Files = []string{}
+
+		for _, f := range []string{"README.md", "LICENCE.md", "LICENSE.md"} {
```

Comment:
I guess it would be nice to get `README`s and `LICENSE`s with other extensions too, so I was thinking about changing this to `README.*`, `LICENSE.*` and `LICENCE.*`...

What do you think?